From kay.sievers@vrfy.org  Thu Jul  2 14:44:01 2009
From: Kay Sievers <kay.sievers@vrfy.org>
Date: Thu, 02 Jul 2009 23:25:44 +0200
Subject: Driver Core: remove BUS_ID_SIZE
To: Greg KH <greg@kroah.com>
Message-ID: <1246569944.1896.2.camel@yio.site>


From: Kay Sievers <kay.sievers@vrfy.org>

The name size limit is gone from the driver-core, this is
the removal of the last left-over.

Signed-off-by: Kay Sievers <kay.sievers@vrfy.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/device.h |    2 --
 1 file changed, 2 deletions(-)

--- a/include/linux/device.h
+++ b/include/linux/device.h
@@ -25,8 +25,6 @@
 #include <asm/atomic.h>
 #include <asm/device.h>
 
-#define BUS_ID_SIZE		20
-
 struct device;
 struct device_private;
 struct device_driver;
